src/main/java/org/generationcp/middleware/service/impl/inventory/PlantingServiceImpl.java

```diff
@@ -62,6 +68,10 @@ public PlantingPreparationDTO searchPlantingPreparation(
 
 		this.processSearchComposite(searchDTO);
 
+		// List Numeric Entry Details variables without ENTRY_NO.
```

Comment:
@cuenyad please remember we are excluding all numeric variables that are system variables (not only entry no)

Reply:
done.